db/migrate/20161010135926_add_content_id_to_specialist_sector.rb

```diff
@@ -0,0 +1,24 @@
2a5-e5372f1e9972", "/topic/population-screening-programmes/newborn-infant-physical-examination"=>"08d40c45-df6c-4827-8d9f-4b638c896fc8", "/topic/work-careers/secondments-with-government"=>"58a8d431-3058-4630-b59f-cdeaf52d6272", "/topic/work-careers/government-graduate-schemes"=>"82ac6e63-4be7-4f30-a207-9d691b4a4405", "/topic/business-enterprise/business-auditing-accounting-reporting"=>"5e16ffb8-694e-4c35-a366-a6cb1cc28a81", "/topic/government/national-and-official-statistics"=>"22e3ce74-bf0c-4647-93c0-eb571355852b", "/topic/business-enterprise/scientific-research-and-development"=>"41f5b331-6fe9-4e14-81b3-374801bb2b35", "/topic/higher-education/scholarships-for-overseas-students"=>"87057249-81f2-4d8c-bb6b-e804abb54de7", "/topic/climate-change-energy/energy-security"=>"f53d65ff-fbbf-456d-bf3c-767323a5b07d", "/topic/climate-change-energy/international-climate-change"=>"06436236-48c9-4335-bd79-60a9e77fe5ac", "/topic/death/death-registration-disclosure"=>"f9bc44f9-0cd7-4cfe-b1dd-f7e217b88a0e", "/topic/climate-change-energy/emissions"=>"1a384903-9171-4252-bb83-b079156207d5", "/topic/climate-change-energy/low-carbon-energy"=>"0d59f5e7-a230-4bac-b02c-624a59a34dab", "/topic/government/public-services"=>"220798e2-ee32-461e-8b22-30a2d03a8d61", "/topic/government/devolution"=>"8e300b1c-de13-4122-b800-5961f3984af0", "/topic/transport/driver-cpc"=>"f4558a4e-f14f-4ec9-85ce-1322e6667d19", "/topic/planning-development/local-plans"=>"2ac2bcbc-cc50-4ae0-a3ea-3ba01c1621e5", "/topic/coal/water-management"=>"0f6e8d9e-96a1-4b7d-8a0a-8f1d525648f2", "/topic/mot/provide-mot-training"=>"eaaebb3e-b059-4624-9092-81b7ece85c5c", "/topic/farming-food-grants-payments/state-aid"=>"0f21c3ff-2c58-4700-826d-14e7285a41e5", "/topic/competition/competition-act-cartels"=>"4a6f14ad-baa1-4b15-8026-8282913ef693", "/topic/mot/manuals"=>"71e36c84-c058-470b-8790-f4c8fd39c615", "/topic/driving-motorcyle-instructors/approved-driving-instructors"=>"bb85c76f-b660-4426-a1c3-37313a2cea81", "/topic/driving-tests-and-learning-to-drive/motorcycle"=>"cb82959c-e637-46aa-8cae-6475ee56b9f2", "/topic/mot/run-mot-test-station"=>"92bb4c52-8c01-463d-a5e1-b841783eff28", "/topic/mot/lorry-bus-trailer-annual-test"=>"2144a80d-3cb0-472c-a0a9-ae738004acf9", "/topic/schools-colleges-childrens-services/opening-academy-free-school"=>"5b8c90ad-acbd-48c5-a39d-af5b841bcd5a", "/topic/business-tax/self-employed"=>"9741abcb-c246-4fd6-bff3-7cee244a9979", "/topic/intellectual-property/copyright"=>"2db5f38b-cf66-455a-bd43-ca4d7c8289df", "/topic/intellectual-property/patents"=>"a98b1005-f153-4012-95cd-1d513b7d28c6", "/topic/intellectual-property/trade-marks"=>"c8e6f8e2-4dac-4ce3-bb02-8690b0e284e4", "/topic/intellectual-property/designs"=>"a949350d-98ed-4260-beb9-f2fc69d15946", "/topic/local-government/byelaws"=>"b01b8809-1fe1-43ff-be0b-8c2581ce6a70", "/topic/driving-motorcyle-instructors/large-goods-vehicle-instructors"=>"5dd90066-ad4b-45c0-b007-cf5bd8cf462e", "/topic/mot/modernisation"=>"020da934-e98c-4925-8e3d-b4fae3fc85f6"}
```

Comment:
Discussed offline... This migration will be removed in a subsequent commit so this is fine.